platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -433,16 +433,16 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark Selecting Annotations
 
 /**
- Returns a Boolean value indicating whether the annotation can be selected.
+ Returns a Boolean value indicating whether the shape annotation can be selected.
  
- If the return value is `YES`, or if this method is absent from the delegate,
- the annotation will be selected.
+ If the return value is YES, the user can select the annotation by tapping [clicking]
```

Comment:
The “clicking” was meant for the macOS implementation of MGLMapView. iOS documentation should never refer to clicking, while macOS documentation should never refer to tapping. (×2)